src/components/CheckboxInput/CheckboxInput.module.css

```diff
@@ -0,0 +1,76 @@
+@import '../../design-tokens/mixins.css';
+
+
+/*------------------------------------*\
+    #CHECKBOXINPUT
```

Comment:
I don't know how BM has been doing it, but can we put a space in here between "CHECKBOX" and "INPUT"? Caps are hard enough to read as it is 😭

Reply:
Definitely could, will add space here and in `CheckboxLabel`